src/components/TodoContext.jsx

```diff
@@ -0,0 +1,35 @@
+import { createContext, useState } from "react"
+import PropTypes from "prop-types"
+
+export const TodoContext = createContext()
+export const TodoProvider = ({ children }) => {
+  const [todos, setTodos] = useState([])
+  const addTodo = (todo) => {
+
+    setTodos([...todos, todo])
+  }
+
```

Comment:
DRY, neat and organized code; making it easy to read and understand.

-Linda&Etna